### DISCUSSION TITLE
Is Multi-Select Annotations Planned for Future Versions?

### DISCUSSION THREAD

**Discussion by gyliarde-involves** [annotorious/annotorious#491] [Q&A]
Hi everyone,

I was wondering if there are any plans to implement multi-select annotations in upcoming versions . This feature would be incredibly useful. If it’s already on the roadmap, could you share any insights on its expected release timeline or priority?

**Comment by rsimon** [comment]
Hi,

let's say it's on the wishlist - and I prepared much of the internal plumbing. Re expected release or timeline: although I'm using Annotorious a lot for my own client projects, it's still a side project. I.e. I can't give any dates or estimates. "When it's done" or "when a paying client needs it" whichever comes first ;-)
